### PR TITLE
run cleanupjob on fly

### PR DESCRIPTION
**Pull request by grooviegermanikus**
run cleanup process in a bash-loop

### CODE REVIEW

File: run-service-and-cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# bring down whole process tree on exit
+trap "kill 0" EXIT
+
+# setup endless loop to run cleanupdb and put it in background
+while true; do
```
Comment (godmodegalactus) [inline run-service-and-cleanup.sh:7]: Should'tn this be a parallel job ?

Comment (godmodegalactus) [reply run-service-and-cleanup.sh:7]: Ahh you said & at the end i guess that makes it parallel job

File: run-service-and-cleanup.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  # startup delay
+  sleep 300;
+  # should retain about 1 week of data
+  RUST_LOG=info /usr/local/bin/cleanupdb --num-slots-to-keep 2000000;
```
Comment (godmodegalactus) [inline run-service-and-cleanup.sh:11]: Change 2000000 to 432000 (epoch) * 2

Comment (grooviegermanikus) [reply run-service-and-cleanup.sh]: updated

**Review by godmodegalactus — APPROVED** [review]

**Pull request merged**